assets/src/design-system/components/dropDown/index.js

```diff
@@ -70,12 +74,23 @@ export const DropDown = ({
       <DropDownSelect
         activeItemLabel={activeOption?.label}
         ariaLabel={ariaLabel}
+        disabled={disabled}
         dropDownLabel={dropDownLabel}
         isOpen={isOpen.value}
         onSelectClick={handleSelectClick}
         ref={selectRef}
         {...rest}
       />
+      {!disabled && (
+        <Popup
+          anchor={selectRef}
+          isOpen={isOpen.value}
+          placement={placement}
+          fillWidth={240}
```

Comment:
I could see us potentially needing to override this. Since most are `240` we can definitely make that the default value.